### ISSUE TITLE
What I Learned in Changeling School Is

### ISSUE THREAD

**Issue by BomberBro** [d3athrow/vgstation13#36019]
### Description of issue
**Changeling.** It sucks. It's not particularly interesting and it's back in rotation so I want to outline a few of its issues as an antagonist and hopefully I can inspire a coder less lazy than I to perhaps take a look at it.

### The Problems:

- If you get husked you can't revive, unless you click the revive button BEFORE you take 200 fire damage. This is somewhat thematic with The Thing but since lasers are ridiculously common and everyone has access to a welding tool or welding fuel, this just in practice makes it impossible to revive unless sec is only using bullets for some reason, but even then that's not bulletproof because:
- If you get your head gibbed or cut off (from, let's say, a shotgun) you can't be revived. This makes your revive ability extra shit even beyond bullets because now if someone knows you can revive anyone with wire cutters or a toolbox to break a window can cut your head off easily in the 1-2 whole minutes it takes to revive yourself, and then you're GONE. It also takes a fucking assload of time to revive which apparently gets lower but never low enough where it's not only ever good as a surprise tactic once. 
- It takes a long fucking time to absorb people, I don't know HOW long exactly but you have to do three really long charge-ups after you already get the guy in an inescapable aggressive choke. I don't think this one is a killer but it'd be nice if it was either made quicker or one of the charge-ups was removed because DAMN this shit takes forever. 
- The create new changeling ability is kind of weird broken and fucked up, you should be able to choose what you want them to look like.
- Changing identities as a ling causes latent genetic disabilities to pop up.
- Most of the abilities are really, REALLY costly like horror form and stuff like deaf-mute+paralysis having to be bought separately, paralysis is already a worse version of a stun because people can still crawl. All the abilities cost stupid amounts of DNA when you consider that absorbing a person doesn't give you all that much, meanwhile vampires only need 3-4 victims to max out their abilities AND can reuse victims for more blood mana. 
 
### Potential Solutions:

- Make changelings get more DNA points from making copies, either that or make any DNA the copies earn also give the OG ling some to encourage replacing crew instead of sneaking around silently murdering the whole crew in your powergame suit one at a time not even using ling abilities. 
- Give changeling some more body changing abilities, TG apparently has a fleshy spacesuit ability to give vacuum resistance, and maybe it'd be cool to be able to make mimic clothes for jobs. Flesh grappling hooks from borers are already ingame and could be nice as well. 
- Let changelings restore missing limbs or heal without their 1-2 minute heal process which KILLS THEM and renders you completely vulnerable. 
- Maybe make gibbing changelings the only way to kill them, but add a blood test to check them in tandem. Considering the chef gibber, cargo/research shuttles, or supermatter shard, it's not THAT hard to figure out a way to kill their ass dead. Throwing them into space naked would work pretty decently too. Fire would be cool to stop their revival but only actual fire and not the current system where you can laser them and they're done. 
- If not, at least make it so changelings can move after being beheaded as a little spider or something and escape into the vents.
- Give changelings 10-20ish messages written by the player they've replaced in their memories so you can figure out how they type and better act as them, give them their bank account number too.

**Comment by PrimeDSS13** [comment]
I agree with all of these statements and options. My main issue is how many people you have to kill for the actually cool abilities
Changeling sucks as is, just a lamer vamp which is impressive!

**Comment by BomberBro** [comment]
Yeah the abilities could really use a cost reduction or give the ling some way of getting more DNA than they are now. I don't mind changelings killing people as long as they have ways to put people back in the round either, it's just that currently they don't even really do that.

**Comment by BomberBro** [comment]
And I'd like to add that part of why changelings act kind of gay as antagonists is because you have really, really lame tools and you can't really take risks and outheal them because there's a solid chance if sec kills you even if you weren't known as a ling you'll get husked or headgored and it's over. All this risk with bad starting tools means you have to play really carefully and can't even think about going loud until you get like 5-6 victims.

**Comment by Killette2** [comment]
Playable headcrab playable headcrab

**Comment by GreatBigFailure** [comment]
I only have two antags disabled, changeling is one of them. The other is disabled because it just forced me to murder/get murdered 5 minutes in. Changeling is disabled because it feels fucking awful to play and I hate it.
